CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
-No changes.
+- Use const generics for pins. The MSRV was bumped to 1.59 ([#316])
```

Comment:
As this pull request changes many things in the gpio API I'd like to have a short list of things which changed like:

* `into_af_push_pull` -> `into_alternate`
* `downgrade().downgrade()` -> `erase()`

Kind of like a short upgrade guide. You don't have to be too detailed. I'm fine to do it myself as well. You can also wait with that change until I finished my thorough review :)